src/js/components/modals/AppModalComponent.jsx

```diff
@@ -72,6 +72,18 @@ var AppModalComponent = React.createClass({
       this.onFieldValidationError);
   },
 
+  componentDidMount: function () {
+    this.setCursorToEndOfAppIdInput();
+  },
+
+  setCursorToEndOfAppIdInput: function () {
+    var appIdInput = React.findDOMNode(this.refs.appId);
+    appIdInput.focus();
+    appIdInput.selectionStart =
+     appIdInput.selectionEnd =
+     appIdInput.value.length;
```

Comment:
Why not use [setSelectionRange](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/setSelectionRange) ?

```
let tot = appIdInput.value.length; 
appIdIinput.setSelectionRange(tot, tot);
```